packages/main/src/themes/Avatar.css

```diff
@@ -9,7 +9,7 @@
 	opacity: .7;
 }
 
-:host(:not([hidden]):not([pressed]):hover) {
+:host([interactive]):not([hidden][pressed][disabled]):hover {
 	box-shadow: var(--ui5-avatar-hover-box-shadow-offset);
 }
```

Comment:
":host(:not([hidden])[pressed]:hover) {
	background: var(--sapButton_Selected_Hover_Background);
	border-color: var(--sapButton_Selected_Hover_BorderColor);
	color: var(--sapButton_Selected_TextColor);
}"

These styles need to be specified for :not([disabled]), interactive state, too.